src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -184,7 +201,7 @@ const NavHeader = (props) => {
           />
         </View>
       )}
-    </Animated.View>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `Animated.View` to `View` may have implications on animations within this component. If there are any animations dependent on this `Animated.View`, they may no longer work as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->